titus-server-runtime/src/main/java/com/netflix/titus/runtime/connector/ChannelTunablesConfiguration.java

```diff
@@ -16,12 +16,14 @@
 
 package com.netflix.titus.runtime.connector;
 
+import com.netflix.archaius.api.annotations.Configuration;
 import com.netflix.archaius.api.annotations.DefaultValue;
 
+@Configuration(prefix = "titus.grpcClientTunnables")
 public interface ChannelTunablesConfiguration {
```

Comment:
Can you rename this file to `GrpcRequestConfiguration`?

Reply:
Yes